README.md

```diff
@@ -1,129 +1,75 @@
-# Azure Functions Plugin 
+# Azure Functions Serverless Plugin 
 
 This plugin enables Azure Functions support within the Serverless Framework.
 
 ## Getting started
 
-### 1. Get a Serverless Service with Azure as the Provider
+### Pre-requisites
 
-1. Recommend using Node v6.5.0
-1. Install the serverless tooling - `npm i -g serverless`
-1. Create boilerplate (change `my-app` to whatever you'd prefer): `serverless install --url https://github.com/azure/boilerplate-azurefunctions --name my-app`
-1. `cd my-app`
-2. `npm install`
+1. Node.js v6.5.0+ *(this is the runtime version supported by Azure Functions)*
+2. Serverless CLI. You can run `npm i -g serverless` if you don't already have it.
+3. An Azure account. If you don't already have one, you can sign up for a [free trial](https://azure.microsoft.com/en-us/free/) that includes $200 of free credit.
 
-### 2. Set up credentials
+### Create a new Azure service
 
-We'll set up an Azure Subscription and our service principal. You can learn more in the [credentials doc]( https://www.serverless.com/framework/docs/providers/azure/guide/credentials).
+1. Create a new service using the standard Node.js template: `serverless create -t azure-nodejs --name <APP_NAME>`
+2. CD into the generated app directory: `cd <APP_NAME>`
+3. Install the app's NPM dependencies, which includes this plugin: `npm install`
+4. Update the `service` property in your `serverless.yml` file, to ensure it specifies a unique app name:
 
-1. Set up an Azure Subscription
-
-    Sign up for a free account @ [https://azure.com](https://azure.microsoft.com/en-us/services/functions/).
-
-    Azure comes with a [free trial](https://azure.microsoft.com/en-us/free/) that includes $200 of free credit. 
-
-
-2. . Get the Azure CLI
-
-    ```
-    npm i -g azure-cli
-    ```
-
-3. Login to Azure
-
-    ```
-    azure login
-    ```
-
-    This will give you a code and prompt you to visit [aka.ms/devicelogin](https://aka.ms/devicelogin). Provide the code and then login with your Azure identity (this may happen automatically if you're already logged in). You'll then be able to access your account via the CLI.
-
-4. Get your subcription and tenant id
-
-    ```
-    azure account show
+    ```yml
+    service: my-azure-functions-app # Name of the Azure function App you want to create
     ```
 
-    Save the subcription and tenant id for later
+### Deploy, test, and diagnose your Azure service
 
-5. Create a service principal for a given `<name>` and `<password>` and add contributor role.
+1. Deploy your new service to Azure! The first time you do this, you will be asked to authenticate with your Azure account, so the `serverless` CLI can manage Functions on your behalf. Simply follow the provided instructions, and the deployment will continue as soon as the authentication process is completed.
 
-    ```
-    azure ad sp create -n <name> -p <password>
+    ```bash
+    serverless deploy
     ```
 
```

Comment:
Add command to deploy one function : serverless deploy function -f <functionName>